scipy/stats/_mstats_extras.py

```diff
@@ -55,6 +55,28 @@ def hdquantiles(data, prob=list([.25,.5,.75]), axis=None, var=False,):
     --------
     hdquantiles_sd
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from scipy.stats.mstats import hdquantiles
+
+    >>> # Sample data
+    >>> data = np.array([1.2, 2.5, 3.7, 4.0, 5.1, 6.3, 7.0, 8.2, 9.4])
+
+    >>> # Define quantiles to compute
```

Comment:
To avoid confusion, I'd suggest referring to these as the "probabilities associated with the quantiles" or similar.

Reply:
Do you want to change the parameter description?

```
prob : sequence, optional
        Sequence of quantiles to compute.
```

to

```
prob : sequence, optional
        Sequence of probabilities associated with the quantiles.
```

Reply:
Yes. In that case, maybe follow the language of `axis`:

> Sequence of probabilities at which to estimate the quantiles.

And use this sort of phrasing (e.g. "Probabilities at which to estimate quantiles" here) in the example.